packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -90,6 +97,9 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 		const el = new window.Image();
 		el.src = url;
 		el.onload = editImage;
+		if ( typeof imageCrossOrigin === 'string' ) {
+			el.crossOrigin = imageCrossOrigin;
```

Comment:
I guess there's no one size fits all value for this? :)

Reply:
Actually, what if we set fixed `use-credentials` here? Is there a scenario where it wouldn't work like that?

Reply:
I don't believe so. The server would need additional configuration to set [`Access-Control-Allow-Credentials`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials). I don't think that's very common for image requests.

It's really something sites need to be able to configure.

Reply:
Not sure I understood. Could you elaborate?

Reply:
The question is whether we can just use a single value that works for everyone. I don't think that will work for all cases, and at the very least could have the undesirable side-effect of sending cookies when you didn't expect or need to.